app/routes/agenda-items/index.ts

```diff
@@ -15,4 +21,22 @@ export default class AgendaItemsIndexRoute extends Route {
       as: 'trefwoord',
     },
   };
+
+  async setupController(
+    controller: Controller,
+    model: unknown,
+    transition: Transition
+  ) {
+    // if mu-search-agenda-items feature is enabled display the mu-search agenda items
+    // otherwise display the default agenda items
+    if (this.features.isEnabled('mu-search-agenda-items')) {
+      console.log(
+        'mu-search-agenda-items feature is enabled, but no visible change yet'
+      );
+      // this.controllerName = 'agenda-items/mu-search';
+      // this.templateName = 'agenda-items/mu-search';
+    }
+
+    super.setupController(controller, model, transition);
+  }
```

Comment:
You can actually, it will be used in https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/83

Reply:
This just seemed like a change that would be part of the mu-search PR (after this one was merged). It added little value to this one, other than an example. I don't think we need the setupController for the real implementation either? Wouldn't we just check the feature flag in the AgendaItemsLoader?

Anyway, we can keep it if you want, but I think it will be removed again later 😄.